code/processes/tailer.q

```diff
@@ -56,3 +56,16 @@ initdatastripe[];
 
 /- create HDB sym file and taildir symlink
 .ds.symlink[];
+
+/- reload the access table and tailDB in the tailreader process
+.tailer.dotailreload[`];
+
+/- define handle to stp and retrieve table stripe mapping
+stphandle:$[count i:.servers.getservers[`proctype;`segmentedtickerplant;()!();1b;0b];
+  first exec w from i;
+    .lg.e[`stphandle;"Failed to get a valid handle to the segmented tickerplant process"]];
+.ds.tblstripe:@[stphandle;"select tbl,filts from .stpps.subrequestfiltered where handle = .z.w";
+    {.lg.e[`.ds.tblstripe;"Failed to retrieve stripe map from STP"]}];
```

Comment:
can this not be stored during subscription, rather than having to send another request to TP?